tools/install-dependencies

```diff
@@ -26,18 +26,23 @@ tar xzf release-$GTEST_VERSION.tar.gz
 
 # Build gtest
 cd googletest-release-$GTEST_VERSION
+echo "patching cmake minimum version"
```

Comment:
Is this needed?  Older versions of cmake are used even by latest gtest?

Reply:
1.10.0 is not latest, gtest prefers live at head (see https://github.com/google/googletest/issues/3124), so I apply the patch based on master code